pkg/controller/kubedirectorcluster/cluster.go

```diff
@@ -103,14 +91,14 @@ func syncCluster(
 		}
 	}()
 
-	// Ignore stale poll-driven handler for a resource we have since
+	// Ignore stale poll-driven reconciler for a resource we have since
 	// updated. Also for a new CR just update the status state/gen.
-	shouldProcessCR := handleStatusGen(cr, handler)
+	shouldProcessCR := handleStatusGen(cr, reconciler)
 
 	// Regardless of whether the status gen is as expected, make sure the CR
```

Comment:
This code comment will need to be tweaked too.

Reply:
Done.